drivers/elum_huawei_smartlogger.json

```diff
@@ -0,0 +1,19 @@
+{
+  "common": {
+    "fncode": 3,
+    "words": 2,
+    "datatype": "float",
+    "order": "lsr"
+  },
+  "fields": {
+    "I_L1": {"register":  2, "unit": "A", "description": "Phase 1 current"},
+    "I_L2": {"register":  4, "unit": "A", "description": "Phase 2 current"},
+    "I_L3": {"register":  6, "unit": "A", "description": "Phase 3 current"},
+    "I_DC": {"register":  8, "unit": "A", "description": "DC current"},
+    "P_DC": {"register":  28, "unit": "A", "description": "DC Power"},
+    "P": {"register":  106, "unit": "W", "description": "AC total active power"},
+    "pf": {"register": 48, "description": "power factor"},
+    "E_Daily": {"register": 110, "unit": "Wh", "description": "power factor"},
```

Comment:
should `Daily` and `Total` be capitalized? I think we usually use lower case. Let's try to standardize on what we've done elsewhere

Reply:
Also `description` here is "power factor" which I assume is the result of some speedy copy-pasting :)

Reply:
Should be fixed too :)